examples/material_gallery/lib/demo/tabs_demo.dart

```diff
@@ -0,0 +1,59 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+import 'widget_demo.dart';
+
+final TabBarSelection _selection = new TabBarSelection();
+final List<String> _iconNames = ["event", "home", "android", "alarm", "face", "language"];
+
+Widget buildTabBar(_) {
+  return new TabBar(
+    selection: _selection,
+    isScrollable: true,
+    labels: _iconNames.map((String iconName) => new TabLabel(text: iconName, icon: "action/$iconName")).toList()
+  );
+}
```

Comment:
I'd made this a member function of _TabsDemoState and declare the parameter.  The idea behind the material_gallery is that it's sample code that people will copy/paste, which means we should try to make the code in it use patterns that we want people to follow.